### DISCUSSION TITLE
Feature Delegations

### DISCUSSION THREAD

**Discussion by thuc-lam** [AnjaliSheshappa/Tafe22S104#1] [General]
Hi guys, so what is everyone doing for the features again?

**Comment by AnjaliSheshappa** [comment]
Hi Thuc,

I'm coding Mortgage calculator, Hope you started your Currency Calculator.

**Comment by AnjaliSheshappa** [comment]
HI Thuc,

Have you finished your coding, If yes, Please push it to the remote repository?
